### PR TITLE
EZP-30363: Provide default Content Tree configuration in meta repositories

### PR DESCRIPTION
**Pull request by webhdx**
> JIRA: https://jira.ez.no/browse/EZP-30363
> Requires: https://github.com/ezsystems/ezplatform-admin-ui/pull/928

### CODE REVIEW

File: app/config/ezplatform.yml [hunk 1]
```diff
@@ -54,6 +54,11 @@ ezpublish:
             # all eng-GB data to other locales first.
             # For admin this needs to contain all languages you want to translate content to on the given repository.
             languages: [eng-GB]
+            content_tree_module:
```
Comment (alongosz) [inline app/config/ezplatform.yml]: @webhdx seems this config node has more required settings, right now with this config I have:
 ```bash
The child node "load_more_limit" at path "ezpublish.system.admin_group.content_tree_module" must be configured.
```

Comment (webhdx) [reply app/config/ezplatform.yml]: It has been fixed in https://github.com/ezsystems/ezplatform-admin-ui/pull/928/commits/fc728868735bdd6864af3192066df9129059c25f

Comment (alongosz) [reply app/config/ezplatform.yml]: > It has been fixed in [ezsystems/ezplatform-admin-ui@fc72886](https://github.com/ezsystems/ezplatform-admin-ui/commit/fc728868735bdd6864af3192066df9129059c25f)

Ah, sorry I didn't see changes in the dependency. It works. Great!

File: app/config/ezplatform.yml [hunk 2]
```diff
@@ -54,6 +54,11 @@ ezpublish:
...
+                contextual_tree_root_location_ids:
+                    - 2 # Home (Content structure)
+                    - 5 # Users
+                    - 43 # Media
```
Comment (micszo) [inline app/config/ezplatform.yml]: How about adding _Setup_ (48)? :)

Comment (webhdx) [reply app/config/ezplatform.yml]: Added in 4af82e3

**Comment by lserwatka** [comment]
@webhdx could you merge it up to -ee-demo?

**Pull request merged**